src/main/java/net/dv8tion/jda/api/events/message/GenericMessageEvent.java

```diff
@@ -71,11 +73,11 @@ public MessageChannel getChannel()
      * @return The GuildMessageChannel
      */
     @Nonnull
-    public GuildMessageChannel getGuildChannel()
+    public GuildMessageChannelUnion getGuildChannel()
```

Comment:
Should we get rid of all the other `getXChannel()` getters on these events?

Reply:
Done